src/main/java/org/opentripplanner/updater/vehicle_rental/VehicleRentalUpdater.java

```diff
@@ -220,8 +213,6 @@ public void run(Graph graph, TransitModel transitModel) {
           latestModifiedEdges.size()
         );
       }
-
-      isPrimed = true;
```

Comment:
Can we put this in a finally of a `try ... catch ... finally` instead, that would make us to wait at least one try?

Reply:
What to you mean with "this"? There is a try/catch but it's in the calling code: https://github.com/opentripplanner/OpenTripPlanner/blob/8ea0a14fde1e0c3f90957ea26ddff6f7407e40af/src/main/java/org/opentripplanner/updater/PollingGraphUpdater.java#L62-L69